src/main/java/org/codeforamerica/messaging/models/SmsMessage.java

```diff
@@ -43,6 +46,6 @@ public class SmsMessage {
     @JsonIgnore
     @OneToOne(mappedBy = "smsMessage")
     private Message message;
+    @JdbcTypeCode( SqlTypes.JSON)
```

Comment:
Extra space, both here and in the matching line of `EmailMessage`.

I'll do a real review next week.

Reply:
Thanks, this is getting to be embarrassing.

Reply:
The minor errors don't matter much and make me feel slightly better when I also miss things :)